File: analyzers/tests/SonarAnalyzer.Test/TestCases/MultipleVariableDeclaration.Fixed.vb
Namespace Tests.Diagnostics

    Public Class MultipleVariableDeclaration
        Public Const AAA As Integer = 5
        Public Const BBB = 42
        Public Const DDD As String = "foo"
        Public Sub New(n As Integer)
            Dim AAA = 1
            Dim B As Integer = 5
            Dim BBB = 42
            Dim CCC As String = "foo"
            Dim DDD As String = "foo"
        End Sub
    End Class
End Namespace

File: analyzers/tests/SonarAnalyzer.Test/TestCases/MultipleVariableDeclaration.Fixed.cs
namespace Tests.Diagnostics
{
    public class MultipleVariableDeclaration
    {
        private int /*aaa1*/ a /*aaa2*/;
        private int
            /*bbb1*/ b /*bbb2*/;

        private int c;
        public MultipleVariableDeclaration(int n)
        {
            int /*aaa1*/a = 0 /*aaa2*/;
            int /*bbb1*/ b = 0 /*bbb2*/;
            int c = 0;
            for (int i = 0, j = 2; i < 3; i++)
            {

            }
        }
    }
}
